homeassistant/components/automation/logbook.py

```diff
@@ -12,9 +12,13 @@ def async_describe_events(hass, async_describe_event):  # type: ignore
     @callback
     def async_describe_logbook_event(event):  # type: ignore
         """Describe a logbook event."""
+        message = "has been triggered"
+        if ATTR_SOURCE in event.data:
+            message = f"{message} by {event.data[ATTR_SOURCE]}"
         return {
             "name": event.data.get(ATTR_NAME),
-            "message": "has been triggered",
+            "message": message,
+            "source": event.data.get(ATTR_SOURCE),
```

Comment:
Logbook isn't localized right now, but some point we might want to add more keys so we can localize.  Probably not for this PR though

`"source_type": "event",  "source_value": "homeassistant_stop"`

Reply:
I guess if we did this now we could show the more info dialog for linked entity for the state change like:

`Master Bed Harmony is Switch Off for 25 seconds has been triggered by state of sensor.watching_tv_in_master_bedroom`

<img width="878" alt="Screen Shot 2020-08-25 at 6 32 24 PM" src="https://user-images.githubusercontent.com/663432/91237876-61cc8b80-e701-11ea-8095-cf3f5f7f89b3.png">

Reply:
Yeah, I realized none of this can be easily localized as is. But I don't know much about that. I leave it to you and/or others to decide if that needs to be addressed as part of this PR or not.

Reply:
Nothing in logbook is localized as is so I don't think it should hold up this PR.

We can add `source_type` and `source_value` in a followup